taichi/transforms/constant_fold.cpp

```diff
@@ -45,8 +45,8 @@ bool TypedConstant::from_binary_op(BinaryOpType op,
   PER_OP(sub, -)
   PER_OP(mul, *)
   PER_OP(div, /)
-  // PER_OP(truediv, /) // XXX: is this same as div?
-  // PER_OP(floordiv, /) // XXX: do we have std::floordiv?
+  PER_OP(truediv, /) // XXX: is this same as div?
```

Comment:
I think `truediv` is something like `(double)a / b` where `a` and `b` are `int`s in C++... Anyway, after we upgrade to JIT, these will become unnecessary :)

Reply:
Maybe we should check return type...if return type is f32, assert it's not truediv.